test/core.layoutService.tests.js

```diff
@@ -230,10 +230,10 @@ describe('Test the layout service', function() {
 		expect(chart.scales.xScale1.right).toBeCloseToPixel(512);
 		expect(chart.scales.xScale1.top).toBeCloseToPixel(484);
 
-		expect(chart.scales.xScale2.bottom).toBeCloseToPixel(28);
+		expect(chart.scales.xScale2.bottom).toBeCloseToPixel(60);
```

Comment:
I think 60px is the correct value because xScale2 is positioned to the top, so we would expect that xScale2.bottom == chartArea.top, which is 60px at line 225.